charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -27,10 +27,12 @@
   name: etc-lsb-release
 {{- end }}
 {{- end -}}
-{{- if eq (include "should-enable-fips" .) "true" }}
+{{- if eq (include "should-enable-fips" .) "true"}}
 - name: fips-config
+  emptyDir: {}
+- name: fips-proxy-cfg
   configMap:
-    name: {{ template "datadog.fullname" . }}-fips-config
```

Comment:
I don't understand why this `configMap` mount was added [prior](https://github.com/DataDog/helm-charts/commit/d7d311157ffe9aeff6b34780063d29fe8a44d43c#diff-e5b9831252dddaaaed4fcb5f16bc06d3aa6871ec8ade6bb7ceb4ae4d09012aadR30-R34) to this PR since we're only now adding the fips proxy configMap object in this current PR. 

Should I retain this behavior to always include the configMap entry if fips is enabled? I'm not sure if there's any negative side effect of keeping this behavior. Or should I only include it if `fips.customFipsConfig` is configured?

Reply:
I think we knew that the need to add this functionality was present but it didn't get implemented right then/there. It might have been useful for debugging too 🤷 . As for when we should include it always or not, since I don't know these workflows too well, I think you would have a better perspective on that decision.